reference/generators/cmake_find_package_multi.rst

```diff
@@ -46,6 +46,7 @@ A target named ``<PKG-NAME>::<PKG-NAME>`` target is generated with the following
 - ``INTERFACE_INCLUDE_DIRECTORIES``: Containing all the include directories of the package.
 - ``INTERFACE_LINK_LIBRARIES``: Library paths to link.
 - ``INTERFACE_COMPILE_DEFINITIONS``: Definitions of the library.
+- ``INTERFACE_COMPILE_OPTIONS``: Compile options of the library.
```

Comment:
this was missing from the documentation although it is taken as an example down below